src/Client/Client.fsproj

```diff
@@ -66,6 +66,13 @@
     <Compile Include="SharedComponents\AdvancedSearch.fs" />
     <Compile Include="SharedComponents\TermSearchInput.fs" />
     <Compile Include="SharedComponents\QuickAccessButton.fs" />
+    <Compile Include="SharedComponents\Generic.fs" />
```

Comment:
What exactly did you do here? The files in maincomponents still exist?

Reply:
They still exist. I wanted to remove them in an extra Issue.

Reply:
fixed